zhmccli/_cmd_partition.py

```diff
@@ -162,6 +162,18 @@ def partition_stop(cmd_ctx, cpc, partition):
 @click.option('--boot-media-file', type=str, required=False,
               help='Boot from removable media on the HMC: The path to the '
               'image file on the HMC.')
+@click.option('--type', type=str, required=False,
+              help='Defines the type of the partition: ssc, linux, zvm.')
```

Comment:
I did not check the API book, but why has partition update the option to change the type and partition create cannot set it?